recipes/blake2/all/makefile

```diff
@@ -0,0 +1,66 @@
+CFLAGS          += $(CONAN_CFLAGS)
```

Comment:
Please, remove the makefile from this project

Reply:
I forgot about it. Sorry about that.

Reply:
No problem, we are almost good to go :)

Reply:
Quick question, on your PR to this recipe, why did you replace the variables _build_subfolder and _source_subfolder whith @properties? Is this the conan-index prefered way?

Reply:
this is to make them read-only and avoid their sudden modification. not that important, but some people prefer that way for safety.

Reply:
I personally dislike this as there is no clear advantage for Conan recipes in using properties for those attributes and they make the recipes more verbose